### ISSUE TITLE
[gltf] Morphing example files ing gltf.js dont morph

### ISSUE THREAD

**Issue by simonharrisco** [pex-gl/pex-renderer#164]
In the gltf.js example file none of the morphing meshes work, they do however work in the gltf-morph.js example file

**Comment by dmnsgn** [comment]
See in gltf-morph.js:
```js
animations.forEach((channels, i) => {
      channels.forEach((channel) => {
```

**Comment by vorg** [comment]
Please add link to the model in gltf-Sample-Models for easy reference of the original glTF file

**Comment by simonharrisco** [comment]
https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/AnimatedMorphCube
![image](https://user-images.githubusercontent.com/36566034/55060588-b893e880-5069-11e9-81f2-d89df107fc02.png)
https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/AnimatedMorphSphere
![image](https://user-images.githubusercontent.com/36566034/55060636-d2cdc680-5069-11e9-8180-7923c9da34b8.png)
https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/MorphPrimitivesTest
![image](https://user-images.githubusercontent.com/36566034/55060670-df521f00-5069-11e9-9adc-98f2cb8b2fd7.png)

and possibly
https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/SimpleMorph
![Uploading image.png…]()

**Comment by simonharrisco** [comment]
most models fixed in #166, 

![image](https://user-images.githubusercontent.com/36566034/55148304-d7fd4500-513f-11e9-8f85-bb3027d28048.png)
https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/MorphPrimitivesTest

Now crashing example file will rename. Possibly related too #175 , will move example to that issue and close this
